test/dumps/plain_zset_2_v11_data.json

```diff
@@ -0,0 +1,6 @@
+"redis-ver":"255.255.255",
+"redis-bits":"64",
+"ctime":"1694535571",
+"used-mem":"965512",
+"aof-base":"0",
+"myzset":{"three":"3.33000000000000007","two":"2","one":"1.1100000000000001"}
```

Comment:
I think zset is probably the best place to test our 64bit read/write, values mainuplation , double2str  and str2double.
Please let's come up with more challenging values, such as,  close to (negative) infinity , close to (negative) zero. and (negative) randomly in the middle that will fail on big-little endian issues.